src/coreComponents/constitutive/permeability/ExponentialDecayPermeability.hpp

```diff
@@ -0,0 +1,163 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ExponentialDecayPermeability.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_PERMEABILITY_EXPONENTIALDECAYPERMEABILITY_HPP_
+#define GEOSX_CONSTITUTIVE_PERMEABILITY_EXPONENTIALDECAYPERMEABILITY_HPP_
+
+#include "constitutive/permeability/PermeabilityBase.hpp"
+
+
+namespace geosx
+{
+namespace constitutive
+{
+
+class ExponentialDecayPermeabilityUpdate : public PermeabilityBaseUpdate
+{
+public:
+
+  ExponentialDecayPermeabilityUpdate( arrayView3d< real64 > const & permeability,
+                                      arrayView3d< real64 > const & dPerm_dPressure,
+                                      arrayView4d< real64 > const & dPerm_dTraction,
+                                      real64 const empiricalConstant,
+                                      R1Tensor const & initialPermeability )
+    : PermeabilityBaseUpdate( permeability, dPerm_dPressure ),
+    m_dPerm_dTraction( dPerm_dTraction ),
+    m_empiricalConstant( empiricalConstant ),
+    m_initialPermeability( initialPermeability )
+  {}
+
+  GEOSX_HOST_DEVICE
+  void compute( real64 const ( &traction )[3],
+                R1Tensor const & initialPermeability,
+                arraySlice1d< real64 > const & permeability,
+                arraySlice2d< real64 > const & dPerm_dTraction ) const;
+
+  GEOSX_HOST_DEVICE
+  virtual void updateFromApertureAndShearDisplacement( localIndex const k,
+                                                       localIndex const q,
+                                                       real64 const & oldHydraulicAperture,
+                                                       real64 const & newHydraulicAperture,
+                                                       real64 const & pressure,
+                                                       real64 const ( &dispJump )[3],
+                                                       real64 const ( &traction )[3] ) const override
+  {
+    GEOSX_UNUSED_VAR( q, oldHydraulicAperture, newHydraulicAperture, dispJump, pressure );
+
+    compute( traction,
+             m_initialPermeability,
+             m_permeability[k][0],
+             m_dPerm_dTraction[k][0] );
+  }
+
+private:
+
+  /// Derivative of fracture permeability to effective normal stress applied on the fracture surfaces
+  arrayView4d< real64 > m_dPerm_dTraction;
```

Comment:
if you want to use this model as fracture permeability we will need to think how to pass this to the FVM kernel. Right now, you will get that `seg fault` because  we are not defining 
````c++
/// Derivative of fracture permeability w.r.t. displacement jump
  array4d< real64 > m_dPerm_dDispJump;
  ````
  
  but it tries to use it here: https://github.com/GEOSX/GEOSX/blob/e9469bf176dbe673e0edd0cb6fbce37d5a4e512c/src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVM.cpp#L458